### ISSUE TITLE
TWE: IS21: RP001: intern Exit Interview Recordings: De-identify: Intern 005

### ISSUE THREAD

**Issue by pandanista** [hackforla/internship#480]
### Dependencies  
~~- [ ] All the user interview issues have been created~~ IS21 didn't set up those issues, but all interviews were conducted. See [TWE: IS21: RP001: Intern Exit Interview Roll Call and Session Table](https://docs.google.com/spreadsheets/d/1dTlIOhtZ5fCNo0GrfKHlOTbdvsRlPUaLIfHk8Cc7Zrw/edit?usp=sharing)
~~- [ ] Interview Recording Upload Form needs to be created~~ The interviews were conducted before we set up the upload form, so it doesn't apply in RP001's case
- [x] Interview recordings have been uploaded and moved into Internship - PII's "My Drive"
- [x] Interview transcripts have been created

### Overview 
We need to clean up and de-identify RP001's Intern Exit interview transcripts so that we can move on to the data analysis and insights generation phase. 

During these tasks, you will be asked to add links to the Resources/Instructions section, and that section is at the bottom of this issue (go view it now).

Another tip is to use split browser windows side by side when working on any GitHub issue, so you can edit the issue on one window, and check the resources or click through links on the other window to avoid a lot of back and forth. Make sure to refresh both windows to reflect any changes you made, including updating the issue or checking the boxes on one of them.

### Action Items
- [x]  UX lead adds the assignee to the Internship - PII's "My Drive" as a Contributor so that they have access to Internship - PII's My Drive where the recordings and transcripts with PII are stored.
   - [x] UX lead accesses the Internship - PII's "My Drive"
   - [x] Choose "Manage members", which is located towards the top right side of the browser <img width="200" alt="Screenshot 2023-08-09 at 6 09 36 PM" src="https://github.com/hackforla/Internship/assets/57029070/f7cc7fbe-1b00-4ded-8b44-b10fdbccae46">
   - [x] Enter the assignee's email address and select the role as "Contributor" <img width="300" alt="Screenshot 2023-08-09 at 6 11 40 PM" src="https://github.com/hackforla/Internship/assets/57029070/41ab3531-75b1-435a-836a-1b24be3ecb69">
   - [x] Confirm with the assignee that they have access to the Internship - PII's "My Drive" 

#### Customize Resource Links
- Customize Resource for Wiki Page Link
    - [x] Go to the wiki page: Research Output Overview (Resources # 1.01)
    - [x] Find relevant wiki page for RP001
    - [x] Copy the link for the wiki page
    - [x] In Resources # 2.01, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [x] Choose "Update comment" in Github and make sure all the checkboxes above have been checked <img width="100" alt="Screenshot 2023-02-21 at 6 47 15 PM" src="https://user-images.githubusercontent.com/57029070/220508425-5d0c0a1b-ef87-4ef2-a683-d4a21d5b2672.png">

- Customize Resource for this Research Plan's Google Drive Folder
    - [x] Open the Google Drive's Research by Type Folder (Resources # 1.02)
    - [x] Choose the `Intern` folder
    - [x]  Choose the `RP001` folder
    - [x] Copy the link of the `RP001` folder
        1. Choose the three vertical dots on the right side of the `RP001` folder  <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/Internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Share"
        1. Choose "Get link"
        1. Choose "Copy Link" and "Done"
    - [x] In Resources # 2.02, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [x] Choose "Update comment" in Github and make sure all the checkboxes above have been checked

- Customize Resource for Interview Recordings and Transcriptions Tracking Sheet
    - [x] Open the Research Plan's Google Drive folder in the Internship's shared drive (Resource # 2.02)
    - [x] Locate the `TWE: IS21: RP001: Intern Exit Interview Recordings and Transcriptions Tracking Sheet` in the folder
    - [x] Copy the link of `TWE: IS21: RP001: Intern Exit Interview Recordings and Transcriptions Tracking Sheet`
        1. Choose the three vertical dots on the top right side of the file  <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/Internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Share"
        1. Choose "Get link"
        1. Choose "Copy Link" and "Done"
    - [x] In Resources # 2.03, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [x] Choose "Update comment" in Github and make sure all the checkboxes above have been checked

- Customize Resource for Interview Recording Folder stored in the Internship - PII's `My Drive`
    - [x] Log into your Google account so you will be able to access the Internship - PII's 'My Drive' in the next steps
    - [x] Choose Shared drives > Internship - PII > Internship -PII
    - [x] Choose `TWE Interview Recordings by Plan #` folder in Internship - PII's `My drive` (Resources # 1.03) 
    - [x] Locate the video recording folder for RP001 inside the folder `TWE Interview Recordings by Plan #`
    - [x] Copy the link of the video recording folder for RP001 
        1. Choose the three vertical dots on the right side of the folder  <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/Internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Share"
        1. Choose "Get link"
        1. Choose "Copy Link" and "Done"
    - [x] In Resources # 2.04, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [x] Choose "Update comment" in Github and make sure all the checkboxes above have been checked
 
- Customize Resource for the De-identified Transcripts Folder in the Shared Drive 
    - [x] Go to RP001 folder (Resources # 2.02)
    - [x] Locate the `RP001 De-identified Transcripts` folder
    - [x] Copy the link of the folder
        1. Choose the three vertical dots on the right side of the folder <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/Internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Share"
        1. Choose "Get link"
        1. Choose "Copy Link" and "Done"
    - [x] In Resources # 2.05, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [x] Choose "Update comment" in GitHub and make sure all the checkboxes above have been checked

- Customize Resources for the Recording and Transcript you are Assigned to
    - [x] Check the title of this issue to identify the participant number, which comes after `De-identify:`
    - [x] Open the `TWE: IS21: RP001: Intern Exit Interview Recordings and Transcriptions Tracking Sheet` (Resources # 2.03)
    - [x] Locate the recording video in .mp4 format that matches the participant number in Column B of the tracking sheet
    - [x] Copy the link of the recording that matches the participant number in Column B of the tracking sheet
    - [x] In Resources # 2.06, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [x] Locate the transcript in .txt format that matches the participant number in Column C of the tracking sheet
    - [x] Copy the link of the .txt file that matches the participant number in Column C of the tracking sheet
    - [x] In Resources # 2.07, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [x] Prepare for cleaning up and de-identification by converting the .txt file into a Google Doc
        1. Click on the link in Column C `Transcription Link` that matches the participant number (Resources # 2.07)
        1. Choose `Open with Google Docs` [![Screenshot 2023-01-06 at 11 04 38 AM](https://user-images.githubusercontent.com/57029070/211081813-b49a22f4-1de2-4a5f-a10a-e7c7686ae1d0.png)](https://user-images.githubusercontent.com/57029070/211081813-b49a22f4-1de2-4a5f-a10a-e7c7686ae1d0.png)
        1. A Google Doc is generated in a new window with the same .txt file name 
        1. The Google Doc is now saved into the same folder with the corresponding video and .txt file
        1. Open the RP001 Interview Recording folder (Resources # 2.04)
        1. Open the corresponding folder for Intern 005 and check to make sure a Google Doc transcript has been created
   - [x] Make a copy of the Google Doc that was just generated and rename it
        1. Choose the three vertical dots on the top right side of the file <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/Internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Make a copy"
        1. A copy of the Google Doc is generated in the same folder       
        1. Copy 
        ```
        To be de-identified-
        ``` 
        1. Go back to the folder where a copy of the Google doc was created
        1. Choose the three vertical dots on the top right side of the file <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/Internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Rename"
        1. Remove `Copy of` from the file name when the Rename text box pops up
        1. Paste what you copied into the beginning of the file name text box
        1.  Choose "Ok"     
    - [x] Copy the link of the to-be de-identified Google Doc transcript you just created
        1. Choose the three vertical dots on the top right side of the file <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/Internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Share"
        1. Choose "Get link"
        1. Choose "Copy Link" and "Done"
    - [x] In Resources # 2.08, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [x] Choose "Update comment" in Github and make sure all the checkboxes above have been checked

#### Clean up and de-identify the transcript
- [x] Open the Google Doc transcript if it is not open yet (Resource # 2.08)
- [x] Listen to the recording to get yourself familiar with the transcript
- [x] Listen to the recording again and clean up the inaccuracies in the transcript as you read along
    - [x] Separate the texts based on speakers and timestamps (timestamps should match the video file timestamps in case a researcher needs to go back and double-check the original video)
    - [x] Add any missed words
    - [x] Edit any typos and misidentified words
    - [x] Deleted repeated words
    - [x] Remove verbal filler words such as "um" and "uh" if they don't serve any purpose
        - Sometimes the verbal fillers can indicate the interviewee's emotions or thoughts. In this case, provide context in square brackets to demonstrate the emotions or thoughts (e.g., [The participant hesitated for a while before coming up with the answer]).
    - [x] Where needed, add context in square brackets so that a reader could understand what was happening without watching the video (e.g., [steps off camera])
- Read through the transcripts again and make edits to de-identify any personal information
    - [x] Search for the participant's name 
    - [x] Replace with their participant number, i.e. I001
    - [x] Search for unknown speakers in the transcript
    - [x] Replace unknown speaker with UXR number or other participant type/number
    - [x] Read through the transcripts to search for any identifiable information in the interview
    - [x] Replace any personal identifiable information with non-identifiable terms. For example, if someone's name is mentioned, we should use their roles inside square brackets so the reader knows this information was redacted (e.g., [team member], [peer], [mentor], [manager]). If their specific school is mentioned, we should redact that info with either [high school] or [college].
    - [x] If they mentioned any specific issues they worked on, make sure to remove the issue numbers and rephrase the issues they worked on
- Read through the edited transcripts again
    - [x] Focus on punctuation, readability, and formatting.
- When you're satisfied that the transcript is completely deidentified
    - [x] Click on the version history and select the most current version
    - [x] Click the three dots <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/Internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96"> and select "Make a copy", and the "Copy version" window pops up
    - [x] In the "Name" text box, remove the "Copy of" text at the beginning and the timestamp at the end from the file name
    - [x] Copy 
    ```
    De-identified Interview Transcript-
   ```
    - [x] Paste the text you just copied in front of the file name in the "Name" text box
        - The new file name should be formatted like "De-identified Interview Transcript-RP001-UX Researcher Abbreviation###-Intern Abbreviation###"
        - An example: "De-identified Interview Transcript-RP006-U007-I001" 
    - [x] Save the renamed de-identified Google doc transcript into the `RP001 De-identified Transcripts` folder. This is to make sure that the version history does not include any identifiable information.
        1. Click the folder icon under "Folder"
        1. Choose "All locations"
        1. Choose Shared drives > Internship > Internships > Research > Research by Type > Intern > folder for RP001 > RP001 De-identified Transcripts
        1. Choose "Select"
        1. Select "Make a copy"
        1. The copy of the file is generated in a new browser
        1. Dismiss the notification that says the file is now in the shared drive
    - [x] Copy the link of the de-identified transcript in the shared drive
        1. Choose "Share"
        1. Choose "Get link"
        1. Choose "Copy Link" and "Done"
    - [x] Paste what you copied into the matching participant's cell in Column D `Transcription Link - de-identified` in the tracking sheet (Resources # 2.03)
    - [x] In Resources # 2.09, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [x] Choose "Update comment" in GitHub and make sure all the checkboxes above have been checked

 
#### 
- [x] Review with UX Lead
- [x] Product sign-off
- [x] UXR Lead or PM removes the assignee from the Internship - PII's My Drive when closing the issue if the assignee no longer needs to access the PII drive.

#### Resources for creating this issue
1.01 [Wiki: Research Output Overview](https://github.com/hackforla/Internship/wiki/Research-Output-Overview)
1.02 [Google Drive Folder: Research by Type](https://drive.google.com/drive/folders/1f5Qgq-ikT_UwcgRBuoBamqY0Wacsg9f5?usp=share_link)
1.03* [TWE Interview Recordings by Plan #](https://drive.google.com/drive/folders/1lvP_k4qGKD05QzbHBtIpEl4B3X7ylzoh?usp=sharing)
1.04 [Guidelines to Interviews Page 8](https://drive.google.com/open?id=19580_-BXfHPC9gTcAj35O2nzAtdNwTxp&disco=AAAA_U2KB_A)
1.04.01 [Brackets vs. Parentheses](https://www.strose.edu/wp-content/uploads/2015/10/Brackets-Handout-Cynthia2.pdf)
1.04.02 [Video folder](https://drive.google.com/drive/folders/1WPzYSrhWYluIqigjsHKpSYIhpQf7XcM5)
1.04.03 [Cleaning Up Zoom Transcriptions for Qualitative Research](https://crdpala.org/2022/10/05/cleaning-up-zoom-transcriptions-for-qualitative-research/)
1.04.04 [Determining Best Practice for Filler Words in Captions and Transcripts](https://blogs.vcu.edu/birdlab/2020/01/09/determining-best-practice-for-filler-words-in-captions-and-transcripts/)

#### Resources gathered during the completion of this issue
2.01 [Wiki: Research Plan RP001](https://github.com/hackforla/internship/wiki/Research-Plan-1:-IS21-Interns)
2.02 [Google Drive Folder: RP001](https://drive.google.com/drive/folders/1uKUY_izpds3dHInNkiexZsU32ELF_Hze?usp=drive_link)
2.03 [TWE: IS21: RP001: Intern Exit Interview Recordings and Transcriptions Tracking Sheet](https://docs.google.com/spreadsheets/d/1Uw_KFUQ3n8DJrxwbc3vnMD8stgF6kTIwSUn5yP9xFuc/edit?usp=sharing)
2.04* [RP001 Interview Recording Folder](https://drive.google.com/drive/folders/1R8sXXGrx47PNj2GpMSUIpMXYllAEgy19?usp=drive_link)
2.05 [RP001 De-identified Transcripts](https://drive.google.com/drive/folders/1A2crr41KWwnoCvdtwmwJnmoP0IzH1Z54?usp=drive_link)
2.06 [Intern 005 Recording Link](https://drive.google.com/file/d/1-8l4ue0SN9hotCgKwhNCGk1foiN-HLWv/view?usp=drive_link)
2.07 [Intern 005 Transcript .txt File](https://drive.google.com/file/d/1yvy8Oo-h_tvsWD7vUILSBeQYovcb-XjT/view?usp=sharing)
2.08 [Intern 005 To be De-identified Transcript Google Doc](https://docs.google.com/document/d/1mu-vdy3j_zjju-vEjcsSe59px5V70TUsrC8S5qOEuz4/edit)
2.09 [Intern 005 De-identified Transcript Google Doc](https://docs.google.com/document/d/1Csyuzm_6i7aoYoiCokKr8c_B_PvXk6E3hIIvgsTHM0Q/edit)

*This folder can only be accessed from the Internship - PII's "My Drive"

**Comment by pandanista** [comment]
Hi @AndrewSalvatore, I assigned intern 005's transcript de-identification issue to you cuz I remember you were working on it.

Also, this issue is based on a new template, so you will need to gather the links again. It might be easier for you to copy from the resources section from #355.

For context, I've split the original transcribe template into two: one for transcribing and setting things up, and the other one is for de-identification. 

Let me know if you have any questions.

**Comment by pandanista** [comment]
@AndrewSalvatore FYI: I've updated the issue above and created a to be de-identified Google doc in the [intern 5 folder](https://drive.google.com/drive/folders/1GfQ_jkQolvP52xt8zqoeg-0QJoAh4vLa?usp=share_link). Thank you for your feedback.

**Comment by AndrewSalvatore** [comment]
Progress: 25% done with transcript cleaning/deidentifying
Blockers: none
Availability: 6 hours
ETA: next Tuesday
Pictures or links (if necessary): none

**Comment by pandanista** [comment]
@AndrewSalvatore Thank you for working on this issue. The transcript looks good. I made some minor edits to clarify a few sentences. It is ready for the Product to review now. 

@sunannie27 Would you please review the deliverable in Resources 2.09? Thank you.

**Comment by sunannie27** [comment]
@pandanista We think it may be better if we color code the interviewer/interviewee portions or create an icon so they can be better distinguished.

**Comment by pandanista** [comment]
@sunannie27 We have decided not to color code the transcript for now since the analysis stage will involve color coding. 

Drew has updated the transcript by incorporating the intern number, please see the updated de-identified transcript linked in Resources # 2.09. Let me know if there is anything else needed. Otherwise, we can close this issue after Product's final review. Thank you.

**Comment by pandanista** [comment]
We need to decide on the format of the timestamps (00:00:00 vs 00:00).

**Comment by pandanista** [comment]
@AndrewSalvatore Welcome back, and hope your 2024 is off to a good start.

Would you please provide an update?
1. Progress: "What is the current status of your project? What have you completed and what is left to do?" - we know you just came back from break, please just summarize the progress from before then
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."

* remember to add links to the top of the issue if they are going to be needed again.

**Comment by pandanista** [comment]
Added a few guidelines on the transcript formatting. You are already following most of them. I am listing them below as a reference.

1. Use the 'in-line' format as seen [here in Intern 001's transcript](https://docs.google.com/document/d/1C8-qEAot7Sw5rhpfuplP0D0SLxypdChUigC1u7LCWJA/edit)
1. Use 6 digit format for all the timestamps, for example, 01:01:01 as hour one/minute one/second one
1. Use "Interviewer" and "Intern ###" throughout the transcript. See an example [here in Intern 001's transcript](https://docs.google.com/document/d/1C8-qEAot7Sw5rhpfuplP0D0SLxypdChUigC1u7LCWJA/edit).
    - If there were more than two people recorded on the transcript, name the third person based on the interview set-up or context, such as Project Manager, Note Taker, etc. If you are not sure what to name them, ask leads on the project to clarify.
1. If there are other numerical values in the transcript, transcribe them using texts. For example, if a time such as 6:30 is mentioned, type out six thirty when cleaning up the transcript. Another example is if the interviewee mentioned he mainly sought help from 3 other peers, type out three instead of using the number 3.

**Comment by sunannie27** [comment]
@AndrewSalvatore 
Please provide update
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."

* remember to add links to the top of the issue if they are going to be needed again.

**Comment by pandanista** [comment]
@AndrewSalvatore Hi Drew, would you please provide an update on this transcript?

I included some more instructions and resources in [comment 1](https://github.com/hackforla/internship/issues/480#issuecomment-1834401961) and [comment 2](https://github.com/hackforla/internship/issues/480#issuecomment-1877618693) above so we can stay consistent when cleaning up the transcript. Let me know if you have any questions.

**Comment by AndrewSalvatore** [comment]
Progress: Reviewed the transcript and it looks like the formatting abides by all the rules mentioned in @pandanista's comments
Blockers: none
Availability: 6 hours
ETA: done
Pictures or links (if necessary): none

**Comment by pandanista** [comment]
Awesome. Thank you, @AndrewSalvatore.

@sunannie27 I reviewed the transcript, and it looks good to me. Would you please review the transcript in Resource 2.09 before we close the issue? Thank you!

**Comment by sunannie27** [comment]
There is a break used with hyphens but hyphens should only be used if there is an emphasis on what was said in the interview, otherwise use commas. 

> I really appreciate you sharing your honest feedback, and your responses were very insightful. So thank you so much for that. I really appreciate you taking the time to share your journey, your experience, your hiccups, and everything else. Thank you. Getting feedback from you - as I mentioned before - is really going to help us improve and enhance the program for our prospective interns. And hopefully it'll enable students interested in STEM programs to learn and move forward with confidence in their workspace. So, I really appreciate your time, and it looks like we're pretty motivated!

**Comment by sunannie27** [comment]
Made the above changes and signed off on it.

**Comment by pandanista** [comment]
Thank you for being so patient with us, @AndrewSalvatore!

Would you please update "interviewer" with "UXR 001" in [intern 005's transcript](https://docs.google.com/document/d/1Csyuzm_6i7aoYoiCokKr8c_B_PvXk6E3hIIvgsTHM0Q/edit?usp=sharing)? Much appreciated!

We decided to include the UXR numbers in the transcripts because 1). there might be different probing questions for semi-structured interviews from different interviewers, and 2). it will be easier to reference the UXR when doing analysis or making highlight reels (if needed).

**Comment by pandanista** [comment]
Thank you very much, @AndrewSalvatore. I am closing this issue since it is all done.

**Comment by pandanista** [comment]
- Reopening this issue since we need to track people being mentioned in the transcripts.

- [ ] UX Research Lead assigns the assignee to the PII drive so they have access to the recording and original transcripts
- Fill out any individual's name and relevant information in the [TWE: IS21: RP001: Participants List from Interviews (Interns & UXR Excluded) spreadsheet](https://docs.google.com/spreadsheets/d/1N76q5QlU8y2O1Eop2iQXWGoQUzTXHQZ2PCDlI6LH2Xc/edit?usp=sharing) by following the steps below:
    - [ ] Open the [TWE: IS21: RP001: Participants List from Interviews (Interns & UXR Excluded) spreadsheet](https://docs.google.com/spreadsheets/d/1N76q5QlU8y2O1Eop2iQXWGoQUzTXHQZ2PCDlI6LH2Xc/edit?usp=sharing) in a new tab
    - [ ] Scroll up to the Resources section to locate recording video and transcripts
    - [ ] Listen to the interview recording in Resources 2.06 to get familiar with the content
    - [ ] Read through the transcript(s) in Resources 2.08 and Resources 2.09 to identify any individual that was mentioned by the intern during the interview
        - Their name in the transcripts might have been replaced by their participant type or number or even spelled wrong, so pay close attention to the transcripts.
    - [ ] Listen to the recording in Resources 2.06 again while reading through the transcript(s) in Resources 2.08 and Resources 2.09   
    - [ ] Identify any individual's name being mentioned, and list out all the relevant information in the spreadsheet. No need to include intern and the UXR names since we track them in the roll call and session table.
        - [ ] Please check each participant type's abbreviation under the "Research Documents by Participant Type" section of [the Research Output Overview Wiki Page](https://github.com/hackforla/internship/wiki/Research-Output-Overview). For example, mentor is M.
        - [ ] The participant numbers are sequential based on the time of entry and occurrence of the mention. I.e. if there are already two Hack for LA website team members being listed previously in the spreadsheet, then the next Hack for LA website team member will be HfLAWTM003.
        - [ ] If the name is already included in the sheet because other interns have mentioned them, please still list out their names and relavent informatioon. Also, for the people who were mentioned repeatedly, please make sure to use the same role and the same participant number that are already assigned to them so we can keep track of the people being mentioned across all interviews.
        - [ ] If you are not sure of the roles of any of the names mentioned, please list the names, timestamps, and transcript links in the spreadsheet, and leave the role column and the participant number column blank. 
            - [ ] Leave a comment to ask for clarification in the transcript by highlighting the individual's name.   
            - [ ] Leave a new comment in this issue or/and inform the UXR lead at the next meeting
            - [ ] UXR lead confirms the role of the individual
    - [ ] After identifying and confirming on the names, roles, and other relevant information, use the search and replace function in Google Doc to replace any identifiable names in the de-identified transcript (link in Resources 2.09) with the participant number assigned to them from the spreadsheet.
- [ ] UXR Lead will review the [TWE: IS21: RP001: Participants List from Interviews (Interns & UXR Excluded) spreadsheet](https://docs.google.com/spreadsheets/d/1N76q5QlU8y2O1Eop2iQXWGoQUzTXHQZ2PCDlI6LH2Xc/edit?usp=sharing)
- [ ] UXR Lead will add the de-identified participants' information to the [TWE: IS21: RP001: De-identified Participants List from Interviews (Interns & UXR Excluded)](https://docs.google.com/spreadsheets/d/1wORsE1sZLtX4T6gbbXqnWEfudJh1ZIE05dE4rpFSoYU/edit?usp=sharing) and save in RP001's folder in the shared Google Drive

Let me know if you have any questions. Thank you.

**Comment by pandanista** [comment]
@floydns: I am assigning this issue to you. 

- Please listen to the interview recording in Resources 2.06, and identify any individuals that were mentioned in the interview session with Intern 002 in the transcripts (Resources 2.08 and 2.09) by listing out their info in a spreadsheet. 
    - The instructions can be found in [this comment](https://github.com/hackforla/internship/issues/480#issuecomment-2138499872). 
    - The names might already be de-identified. For example, project lead or lead mentor might be used in the de-identified transcript. In that case, list out the name (based on the interview recording), role, and other info in the spreadsheet. If the name wasn't there in the spreadsheet, assign them a new participant number based on participant type. If the name already exists in the spreadsheet, then use the same participant number they are already assigned to. 
    - Please check off the boxes in [the comment above](https://github.com/hackforla/internship/issues/480#issuecomment-2138499872) as you go along.

An example of a similar issue can be found [here](https://github.com/hackforla/internship/issues/481#issuecomment-2138505878).

Thank you very much!

**Comment by floydns** [comment]
@pandanista I believe I can finish this over the next few weeks. Just slammed at the moment :)

**Comment by pandanista** [comment]
@floydns Thank you for the update. Let me know if you have any questions.

**Comment by pandanista** [comment]
I am re-assigning the issue to @jinyan0425.

- Please listen to the interview recording in Resources 2.06, and identify any individuals that were mentioned in the interview session with Intern 002 in the transcripts (Resources 2.08 and 2.09) by listing out their info in a spreadsheet. 
    - The instructions can be found in [this comment](https://github.com/hackforla/internship/issues/480#issuecomment-2138499872). 
    - The names might already be de-identified. For example, project lead or lead mentor might be used in the de-identified transcript. In that case, list out the name (based on the interview recording), role, and other info in the spreadsheet. If the name wasn't there in the spreadsheet, assign them a new participant number based on participant type. If the name already exists in the spreadsheet, then use the same participant number they are already assigned to. 
    - Please check off the boxes in [the comment above](https://github.com/hackforla/internship/issues/480#issuecomment-2138499872) as you go along.

An example of a similar issue can be found [here](https://github.com/hackforla/internship/issues/481#issuecomment-2138505878).

Thank you very much!

**Comment by pandanista** [comment]
@jinyan0425: Thank you for highlighting and commenting on the doc. I have reviewed them, and updated or corrected developers' names in [TWE: IS21: RP001: Participants List from Interviews (Interns & UXR Excluded) spreadsheet](https://docs.google.com/spreadsheets/d/1N76q5QlU8y2O1Eop2iQXWGoQUzTXHQZ2PCDlI6LH2Xc/edit?usp=sharing).

Everything looks great. 

I have also added the relevant info to [TWE: IS21: RP001: De-identified Participants List from Interviews (Interns & UXR Excluded)](https://docs.google.com/spreadsheets/d/1wORsE1sZLtX4T6gbbXqnWEfudJh1ZIE05dE4rpFSoYU/edit?usp=sharing). 

We can close this issue now.
